### ISSUE TITLE
idea: disable block duplicating

### ISSUE THREAD

**Issue by psyafter** [voidlabs/mosaico#108]
Nice to have option to disable block duplicating.
If there is special property on block so this block can be used only once.
Without option to add another same block and prevent duplicating this block.

**Comment by bago** [comment]
In the "email composing" world, I don't think this is so general purpose to be implemented.

You start with this requirement, then you need some block to be present 1 or 2 times, some other block to be only after another or to be placed only if you have already another one... there is no clean/right path to follow and this make the code more complex and the library not much more flexible.

**Comment by psyafter** [comment]
understood you.
instead of this - there is internal option to disable "blocks" tab?
